pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -136,29 +136,30 @@ func (c *Cluster) Start() error {
 				return fmt.Errorf("failed to update mgr deployment %s. %+v", resourceName, err)
 			}
 		}
-	}
```

Comment:
I don't think we want to add all the following calls to the `for` loop. If we ever started two mgr pods (`a` and `b`), we would still only want a single service, enable the modules once, etc. If we need the dashboard settings per mgr pod, I would think only that needs to be in the `for` loop.

Reply:
Puh, to be honest i don't know... i wrote it this way b/c that way worked for me...